aspnet/hosting/directory-structure.rst

```diff
@@ -3,17 +3,42 @@
 Directory Structure
 ===================
 
-By `Sourabh Shirhatti`_
+By `Luke Latham`_
 
 
-In ASP.NET Core, the application directory comprises of three sub-directories. This is unlike previous versions of ASP.NET where the entire application lived inside the web root directory. The recommended permissions for each of the directories are specified in the table below.
+In ASP.NET Core, the application directory, *publish*, is comprised of application files, config files, static assets, packages, and the runtime (for self-contained apps). This is unlike previous versions of ASP.NET, where the entire application lived inside the web root directory.
```

Comment:
@shirhatti I'm trying to avoid a one sentence paragraph there. [I used to mark my students' papers down when they did that! lol] I could flip that sentence to ...

> This is the same directory structure as previous versions of ASP.NET, where the entire application lives inside the web root directory.

... or possibly just mention the the directory structure is slightly diff between the two app types ...

> The difference in directory structure between a portable application (the default app type) and a self-contained application is that the portable application will include a runtimes directory if native dependencies are required and the application itself will be a DLL file. A self-contained application will publish as an EXE file with its runtime and other DLL files to make it capable of running without any shared components present on the server.